microsoft/testsuites/lis/lissuite.py

```diff
@@ -50,7 +50,7 @@ def verify_lis_driver_version(self, node: Node, log: Logger) -> None:
         cat = node.tools[Cat]
         modinfo_tool = node.tools[Modinfo]
         wget_tool = node.tools[Wget]
-        node.tools[LisDriver]
+        self._install_lis(node)
```

Comment:
The method is to check lis supported or not, not to install it. So it should be called like `_check_lis_installable`.

Reply:
updated